scaleway/resource_server.go

```diff
@@ -41,6 +41,19 @@ func resourceScalewayServer() *schema.Resource {
 				Description:  "The instance type of the server",
 				ValidateFunc: validateServerType,
 			},
+			"boot_type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				Description:  "The boot_type of the server",
+				ValidateFunc: validateBootType,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Given this value would have been set to “bootscript” in the past, I think we can remove this in favour of a default value or a State migration which sets a default?

Reply:
Infact - is this computed?

Reply:
come to think about it, Computed and Optional I'd say. I'll adjust accordingly.